O/obstack/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "obstack"
+version = v"1.1.1"
```

Comment:
Shouldn't this be
```suggestion
version = v"1.1.0"
```
?

Reply:
Oh yeah probably, I just copied the patch version from Alpine, but that's probably just an Alpine-specific version.